docker/tidb-sysbench/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM alpine:3.9
+MAINTAINER PingCAP <cloud@pingcap.com>
+
+# Install sysbench
+# gcc and mariadb-dev install shared libraries used at runtime
+RUN apk add gcc mariadb-dev
+
+RUN apk add --virtual .build-deps git build-base automake autoconf libtool --update \
+  && git clone https://github.com/akopytov/sysbench.git \
+  && cd sysbench \
+  && git checkout 1.0.17 \
+  && ./autogen.sh \
+  && ./configure --disable-shared \
+  && make \
+  && make install \
+  && cd .. \
+  && rm -r sysbench \
+  && apk del .build-deps
+
+# TiDB patch for sysbench data loading (prepare)
+RUN apk add --virtual .build-deps wget \
```

Comment:
Why run `apk add` again? A `wget` to download the oltp_common.lua is enough.

Reply:
I think it is easy to `apk add` again if you need it later.